index.html

```diff
@@ -244,7 +244,25 @@ <h2 class="Frameworks">Skills</h2>
         </div>
       </section>
         </div>
-      
+        <!--Contact Form Begins-->
+
+       
+    <div class="contact-form">
+        <h2>Contact Me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form action="https://formspree.io/f/mrgjdedp"
+        method="POST">
+          <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required><br>
+          <input type="email" id='email' name="email" placeholder="email"   required><br>
+          <textarea  placeholder="Write your message here" name="message" id="message" cols="30" rows="10"  maxlength="500" required></textarea><br>
+          <button type="submit">Get In Touch</button>
```

Comment:
- Optional
Instead of using the `<br>` tag, it is better to use the `display: flex` on the `from` element itself. That would make more sence. Don't you think so?